extensions/amp-story/1.0/amp-story.css

```diff
@@ -684,6 +684,7 @@ amp-story .amp-video-eq,
   animation-timing-function: cubic-bezier(0.4, 0.0, 0.2, 1) !important;
   animation-iteration-count: infinite !important;
   opacity: 1 !important;
+  filter: drop-shadow(0px 0px 4px rgba(0,0,0,0.35)) !important;
```

Comment:
The blur radius is very high and the result looks... blurry :))

Can you try other numbers, like `0 1px 3px 1px rgba(0, 0, 0, 0.12)` maybe?

Reply:
Done.